.github/workflows/diff_prs.yml

```diff
@@ -0,0 +1,53 @@
+name: Diff as PRs
+
+on:
+  workflow_dispatch:
+    inputs:
+      base_branch:
```

Comment:
Do we really need 2 inputs ? 

We could use the branch the job runs on (like we do for experimentals) as the ref. here we would always select a release branch.

And then we need the base branch that would be the previous version tag. 

do we have a way to make this into a select ? and side not do you support tags the same way you support branches (it's supposed to just be refs but just in case you need to test it)

Reply:
The goal of having two inputs was to allow for more precise diffs if needed (e.g. to track down where a bug is originating), I was hoping the input would be a select by default just like the publish/pre-release actions. (I've tried looking them up for specific options but haven't found anything.) If you have tips on that, they're welcomed :smiley: 

We could pass a default though

Yes it supports tags (which is super useful), I've been testing the script between `v5.10.3` and `releases/5.10.4` and it was working fine

Reply:
Yeah that's what I was expecting sadly I don't know a way to dynamically display a list of branch in a select :/